packages/manager/apps/pci-load-balancer/src/pages/create/steps/region/RegionStep.tsx

```diff
@@ -131,7 +134,12 @@ export const RegionStep = ({
       order={1}
       next={{
         action: () => {
-          trackStep(1);
+          trackClick({
+            actions: ['select_location', `add_${store.region?.name}`],
```

Comment:
When clicking next the region shouldn't be null isn't it ?

Reply:
So if at this point it cannot be null the '?' shouldn't be set, you can use '!' instead to assert the nun-null